### PR TITLE
Add `addFlowTopic` function to bot class

### LINKED ISSUE #117: Add new-topic for Flow support
It would be usefull to add the `new-topic` for the "new" Flow extension for talk pages.

Documentation: https://www.mediawiki.org/w/api.php?action=help&modules=flow%2Bnew-topic

### PR DESCRIPTION
**Pull request by ValentinBrclz**
Add `addFlowTopic` function to bot class

Feedback and correction requests are welcome.

This closes #117

### CODE REVIEW

File: lib/bot.js [hunk 1]
```diff
@@ -558,6 +558,46 @@ module.exports = (function() {
...
+					token: token
+				};
+
+				self.api.call(params, function(err, data) {
+					if (!err && data['new-topic'] && data['new-topic'].status && data['new-topic'].status === "ok") {
+						self.log("Workflow '%s' created on '%s'", data['new-topic'].workflow, title);
+						callback(null, data);
+					}
+					else {
+						callback(new Error(`Edit failed: ${err}`));
```
Comment (macbre) [inline lib/bot.js]: I'd simply pass `err` (without creating a new exception object) to the callback to be consistent with the rest of the code.

**Review by macbre — CHANGES_REQUESTED** [review]

File: lib/bot.js (hunk 1 above)
Comment (ValentinBrclz) [reply lib/bot.js]: Sure, I took example on the `doEdit` function, this is why it is here. Will fix it asap

Comment (macbre) [reply lib/bot.js]: :+1: 

And I'll correct the rest of the cases where `err` is wrapped within a new exception instance - #121

**Review by macbre — APPROVED** [review]

**Comment by macbre** [comment]
Thanks!

**Pull request merged**